Project.toml

```diff
@@ -11,11 +11,11 @@ Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Tokenize = "0796e94c-ce3b-5d07-9a54-7f471281c624"
 
 [compat]
-CSTParser = "^3.2.3"
+CSTParser = "^3.2.4"
```

Comment:
Do we need to bump the package version too?

Reply:
https://github.com/domluna/JuliaFormatter.jl/pull/448/commits/8c274fff8661055df3a6aa83f2c2802938ea2808